src/main/resources/initdb.sql

```diff
@@ -26,6 +26,7 @@ CREATE TABLE IF NOT EXISTS t_product (
 	min_order_amount DECIMAL(32,18) NOT NULL,
 	max_order_amount DECIMAL(32,18) NOT NULL,
 	min_order_funds DECIMAL(32,18) NOT NULL,
+    enable BOOLEAN NOT NULL DEFAULT TRUE,
```

Comment:
How about to rename to `disabled`, which is `false` by default.

Reply:
ok